api/src/app/modules/analytics-dashboard/analytics-dashboard.controller.ts

```diff
@@ -0,0 +1,217 @@
+} from '@api-modules/public-comment/public-comment.dto';
+
+@ApiTags('analytics-dashboard')
+@Controller('analytics-dashboard')
+export class AnalyticsDashboardController {
+  constructor(private readonly analyticsDashboardService: AnalyticsDashboardService) {}
+
+  /**
+   * Returns the total number of FOM projects submitted
+   * within the specified commenting open date range,
+   * excluding those with an INITIAL workflow status.
+   *
+   * @param query - DateRangeRequest containing startDate and endDate in 'YYYY-MM-DD' format
+   * @returns Number of FOM projects in the date range
+   */
+  @Get('project/count')
+  @ApiOperation({
+    summary: 'Get total number of FOMs submitted in a date range',
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'Total number of FOMs',
+    type: Number,
+  })
+  async getProjectCount(@Query() query: DateRangeRequest): Promise<number> {
+    return this.analyticsDashboardService.getProjectCountByDate(
+      query.startDate,
+      query.endDate
+    );
+  }
+
+  /**
+   * Returns the number of FOM projects grouped by district
+   * within the specified commenting open date range,
+   * excluding those with an INITIAL workflow status.
+   *
+   * @param query - DateRangeRequest containing startDate and endDate in 'YYYY-MM-DD' format
+   * @returns An array of objects containing districtId, districtName, and projectCount
+   */
+  @Get('project/count-by-district')
+  @ApiOperation({
+    summary: 'Get total number of FOMs grouped by district in a date range',
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'List of FOM project counts by district',
+    type: ProjectCountByDistrictResponse,
+    isArray: true,
+  })
+  async getProjectCountByDistrict(
+    @Query() query: DateRangeRequest
+  ): Promise<ProjectCountByDistrictResponse[]> {
+    return this.analyticsDashboardService.getProjectCountByDistrict(
+      query.startDate,
+      query.endDate
+    );
+  }
+
+  /**
+   * Returns the total number of unique forest clients who submitted FOMs
+   * within the specified commenting open date range,
+   * excluding those with an INITIAL workflow status,
+   * and those with null forest client numbers.
+   *
+   * @param query - DateRangeRequest containing startDate and endDate in 'YYYY-MM-DD' format
+   * @returns Number of distinct forest client numbers
+   */
+  @Get('project/count-forest-client')
+  @ApiOperation({
+    summary:
+      'Get the number of unique forest clients who submitted FOMs in a date range',
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'Total unique forest client count for submitted FOMs',
+    type: Number,
+  })
+  async getUniqueForestClientCount(
+    @Query() query: DateRangeRequest
+  ): Promise<number> {
+    return this.analyticsDashboardService.getUniqueForestClientCount(
+      query.startDate,
+      query.endDate
+    );
+  }
+
+  /**
+   * Returns the number of FOM projects submitted by each forest client
```

Comment:
Same comment as above with regards to submitted versus published.

Reply:
This one is for number of FOMs submitted by each Forest Client. Should it be Returns the number of FOM projects by each forest client with a commenting open date within the specified date range, excluding those with an INITIAL workflow status? Or we want to filter the data in another way?